### ISSUE TITLE
1 processed stylesheet created

### ISSUE THREAD

**Issue by 58bits** [nDmitry/grunt-postcss#37]
Great grunt task so thanks for this.

Not sure if this is designed behavior or not, but I have three .css stylesheets the public/css/ directory, and all three are being processes, however, the console output only ever says '1 processed stylesheet created'.

```
postcss: {
      options: {
        map: false,
        processors: [
          require('pixrem')(), // add fallbacks for rem units
          require('autoprefixer-core')({browsers: 'last 2 versions'}),
          require('cssnano')() // minify the result
        ]
      },
      dist: {
        src: 'public/css/*.css'
      }
    }
```

**Comment by hudochenkov** [comment]
Confirm. I have only autoprefixer-core processor.

**Comment by nDmitry** [comment]
Thanks, fixed in 0.5.5
